Nvidia_GPU_Temperature.py

```diff
@@ -14,19 +14,20 @@
 
 while True:
 
-    output = subprocess.check_output(["C:\\Program Files\\NVIDIA Corporation\\NVSMI\\nvidia-smi.exe", "-a"], shell=True)
+    output = str(subprocess.check_output(["C:\\Program Files\\NVIDIA Corporation\\NVSMI\\nvidia-smi.exe", "-a"], shell=True))
     #os.popen('C:\\Program Files\NVIDIA Corporation\NVSMI\nvidia-smi.exe')
     #output=os.popen("C:\\Program Files\\NVIDIA Corporation\\NVSMI\\nvidia-smi.exe").read()
-    
+
     #print("====" + str(output) + "=====")
     temp = re.search("GPU Current.*",output).group()[30:33]
-    temp_baseline = 60
+
+    temp_baseline = 55  # configure to you base line temp here
     temp_multiplier = 5
     color_temp = (int(temp) - temp_baseline ) * temp_multiplier
-    green = 100 - color_temp
-    red = 0 + color_temp
+    green = min(254, max(0,100 - color_temp))
+    red = min(254, max(0,0 + color_temp))
```

Comment:
pep8: have a space after the comma.

Reply:
I wasn't really focused on formating there are a lot of other style violations in the codebase. I might do another round sometime soon.